k8s/chainctl

```diff
@@ -180,7 +180,7 @@ files_with_same_basename() {
   local file=$(basename "$1")
   local type="${file##*.}"
   local basename="$(basename "$file" "$type")"
-  find . -maxdepth 1 -iname "$basename*" -not -name $file
+  find . -maxdepth 1 -iname "$basename*" -not -name $file -not -name "cl-sslcert*"
```

Comment:
I do rename the output for `chainctl certificates` to match this naming convention.

Reply:
Yeah but that won't stop you from using any other names for certificate and the `cl-` name only makes sense for chainlink (`cl` is an abbreviation). I think it would be better to do something like:

```bash
  files=$(find . -maxdepth 1 -iname "$basename*" -not -name $file)
  case type
    crt)
      files=$files | grep -v key
```

(pseudocode)